packages/react-ui-kit/src/Form/Checkbox.jsx

```diff
@@ -17,10 +17,13 @@
  *
  */
 
+import {COLOR} from '../index';
 import {Input} from './Input';
 import PropTypes from 'prop-types';
 import React from 'react';
+import {Text} from '../Text';
 import styled from 'styled-components';
+import {v4 as uuid} from 'uuid';
```

Comment:
Do you really need a `uuid` in a styled component? To me UUID sounds like business logic. Nothing that you would want to important in a presentation layer.

Reply:
Maybe not. Actually I just need some random string. Something like Math.random().toString() would work as well.

Reply:
👍 , then we can also remove 'uuid' from the dependencies?